Riot/Modules/Common/Recents/RecentsViewController.m

```diff
@@ -1130,6 +1130,11 @@ - (UITableViewCellEditingStyle)tableView:(UITableView *)tableView editingStyleFo
 
 - (nullable UISwipeActionsConfiguration *)tableView:(UITableView *)tableView trailingSwipeActionsConfigurationForRowAtIndexPath:(NSIndexPath *)indexPath
 {
+    if (BuildSettings.newAppLayoutEnabled)
+    {
+        return nil;
```

Comment:
It has been decided with @amshakal that we definitively get rid of the swipe menu action. If this can relieve your grief, leave room action is also available from the context menu with a nice destructive style :)

Reply:
Wooo 🥳